shell/config/router/navigation-guards/clusters.js

```diff
@@ -71,8 +31,12 @@ export default async function({
     const oldPkg = getPackageFromRoute(from);
     const oldProduct = getProductFromRoute(from);
 
+    // TODO: Replace all references to store.$plugin.
```

Comment:
Thanks for clarifying the change in the comment and sorry for asking again, could you please add an issue for this?

Reply:
Since I'll have to migrate this part, may I ask you where we assign the global `$plugin` to the store? Or is it something from Vuex?

Reply:
Here's the issue: https://github.com/rancher/dashboard/issues/11386

I've added a reference to where `$plugin` is set in the tech debt issue but here's a reference here as well https://github.com/rancher/dashboard/blob/5a00bcc07cd7cf724fa6ca0d1ca7dff57108b23f/shell/store/index.js#L1149

Reply:
I add it to the Nuxt removal bits, since it was listed 😅 
https://github.com/rancher/dashboard/issues/11070